packages/forms-web-app/src/dynamic-forms/has-appeal-form/journey.js

```diff
@@ -37,6 +37,16 @@ class HasAppealFormJourney extends Journey {
 				.withCondition(
 					response.answers && response.answers[questions.ownsAllLand.fieldName] == 'no'
 				)
+				.addQuestion(questions.ownsRestOfLand)
+				.withCondition(
+					response.answers && response.answers[questions.ownsSomeLand.fieldName] == 'yes'
+				)
+				.addQuestion(questions.ownsLandInvolved)
+				.withCondition(
+					response.answers &&
+						response.answers[questions.ownsSomeLand.fieldName] == 'no' &&
+						response.answers[questions.ownsAllLand.fieldName] == 'no'
+				)
```

Comment:
we actually have a fancy thing for doing these, if you want to use it check out `questionsHaveAnswersBuilder` that's used in the s78 journey

Reply:
Ah didnt know about these, have added thanks. 

The next step of the journey will be fun with there being multiple combinations/ possibilities for the same pages!